src/main.js

```diff
@@ -0,0 +1,87 @@
+const myLibrary = [];
+
+function Book(author, title, pages, read) {
+  this.author = author;
+  this.title = title;
+  this.pages = pages;
+  this.read = read;
+}
+function setStatus(read, button) {
+  if (read) {
+    button.innerText = 'Read';
+  } else {
+    button.innerText = 'Not read yet';
+  }
+}
+
+function switchStatus(e) {
+  const index = e.srcElement.attributes['1'].value;
```

Comment:
`srcElement` is deprecated, you should use `target`: https://developer.mozilla.org/en-US/docs/Web/API/Event/srcElement

And instead of using `attributes["1"].value` you could try `getAttribute("rel")`. It's is shorter and more readable.